Makefile

```diff
@@ -103,8 +103,13 @@ test: vendor $(GOPATH)/src/$(ORG)
 	@go test -v $(shell $(PACKAGES))
 
 .PHONY: integration
+ifdef BINPATH
```

Comment:
@hferentschik Thanks for tips, I really like your idea. I am just not sure with naming of variable - there is conflict between clarity and friendliness I think. For me `make integration MINISHIFT_INTEGRSTION_TEST_BINARY=~/minishift/minishift` feels unfriendly, also when we add tags support then it will be even longer. WDYT for example about `TEST_BINARY`?